otcextensions/sdk/ces/v1/_proxy.py

```diff
@@ -1,17 +1,146 @@
-# Licensed under the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License. You may obtain
-# a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-from openstack import proxy
-
-
-class Proxy(proxy.Proxy):
-
-    skip_discovery = True
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from openstack import proxy
+
+from otcextensions.sdk.ces.v1 import alarm as _alarm
+from otcextensions.sdk.ces.v1 import event_data as _event_data
+from otcextensions.sdk.ces.v1 import metric as _metric
+from otcextensions.sdk.ces.v1 import metric_data as _metric_data
+from otcextensions.sdk.ces.v1 import quota as _quota
+
+
+class Proxy(proxy.Proxy):
+
+    skip_discovery = True
+
+    # ======== Alarms ========
+    def alarms(self, **query):
+        """Return a generator of alarms

```

Comment:
Missing info about query params -- 
:param dict query: Optional query parameters to be sent to limit the resources being returned.